Add README (#4)
radicle-dev/radicle-avatar: README.md

# radicle-avatar

> A library for generating emoji-color combinations which serve as avatars on the
> Radicle network.

Reviewer: ```suggestion
A library for generating emoji-color combinations which serve as avatars in
Radicle clients.
```

The network/protocol doesn't have a notion of avatars.
